sqlitecluster/SQLiteNode.h

```diff
@@ -191,6 +191,8 @@ class SQLiteNode : public STCPNode {
     void setQuroumCheckpoint(const int quroumCheckpoint) { _quorumCheckpoint = quroumCheckpoint; };
     int getQuorumCheckpoint() { return _quorumCheckpoint; };
 
+    bool isQueuedCommandMapEmpty();
+
```

Comment:
A nitpick, but can you add a comment above this like:

```
// Checks if we have any commands ready to process
```

In fact... now that I write that, I realize that I think we're using that command wrong.  We're using the existence of commands in the queue to signal that work is ready to go, but that's actually not true.  The commands might be held, or have outstanding `httpsRequests`, or be scheduled for the future, etc.  In this situation, we'd go into a busy loop where we check the network, but with an instant timeout thinking we're going to do something -- even though each time we conclude we have nothing to do.

I'd suggest either copying the logic for determining if we *actually* have work to do, or even updating this to return an actual iterator to the command to process -- such that we keep all this logic in one place.

Either way, let's also rename this to something like `getNextCommand()`